DeviceManager/DatabaseModels.py

```diff
@@ -190,3 +194,24 @@ def receive_set(target, value, old_value, initiator):
     if old_value == 'psk' and value != 'psk':
         for key in target.pre_shared_keys:
             db.session.delete(key)
+
+@event.listens_for(DeviceAttr, 'load')
+def transform_get(target, context):
+    if target.type == "static":
+        # Checking for JSON compatible types
+        if target.value_type == "number":
+            target.static_value = int(target.static_value)
+        elif target.value_type == "string":
+            # static_value is already a string
+            pass
+        elif target.value_type == "boolean":
```

Comment:
In GUI we use, maybe you can check if is 'boolean' or 'bool'
`  this.availableValueTypes = [
            { value: 'geo:point', label: 'Geo' },
            { value: 'float', label: 'Float' },
            { value: 'integer', label: 'Integer' },
            { value: 'string', label: 'String' },
            { value: 'boolean', label: 'Boolean' },
        ];
        this.availableTypes = [
            { value: 'dynamic', label: 'Dynamic Value' },
            { value: 'static', label: 'Static Value' },
            { value: 'actuator', label: 'Actuator' },
        ];
        this.configTypes = [
            { value: 'mqtt', label: 'MQTT' },
        ];
        this.configValueTypes = [
            // {"value": "fw_version", "label": "Firmware Version" },
            { value: 'protocol', label: 'Protocol' },
            { value: 'topic', label: 'Topic' },
            { value: 'translator', label: 'Translator' },
            { value: 'device_timeout', label: 'Device Timeout' },
        ];`

Reply:
I'll add a 'bool' check.

Reply:
Besides of that now we need to add object

Reply:
I'll take a look

Reply:
`object` should be already supported - it is processed the same way as an array.